### PR TITLE
Add total Pokemon counter to pokedex.page.php

### PR DESCRIPTION
**Pull request by SiteEffect**
Please test. Works for me.

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

**Comment by SiteEffect** [comment]
Can't figure out why 🐶 doesn't like me

File: core/process/data.loader.php [hunk 1]
```diff
@@ -211,6 +211,11 @@
 			// Pokemon List from the JSON file
 			// --------------------------------
 
+			$req			= "SELECT TABLE_ROWS as total FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'pokemon' AND TABLE_SCHEMA = '".SYS_DB_NAME."'";
+			$result         = $mysqli->query($req);
+        	$data           = $result->fetch_object();
+
+        	$total	        = $data->total;
```
Comment (Obihoernchen) [inline core/process/data.loader.php]: Indentation no wau wau :dog:

File: pages/pokedex.page.php [hunk 2]
```diff
@@ -4,7 +4,12 @@
 			<h1>
 				<?= $locales->POKEDEX_TITLE ?>
 			</h1>
-
+			<h3>
+				<?= sprintf($locales->POKEDEX_TOTAL, number_format($total), $config->infos->city) ?>
+			</h3>
+			/*<p>
+				<?= $locales->POKEDEX_ULTRA_COMMONS ?>
+			</p>*/
```
Comment (Obihoernchen) [inline pages/pokedex.page.php]: What's that? It's not a valid comment in html.

**Review by Obihoernchen — CHANGES_REQUESTED** [review]

**Comment by Obihoernchen** [comment]
I removed the POKEDEX_ULTRA_COMMONS string for now. We might add this in the future but in a proper way ;)
Looks really cool 👍

**Pull request merged**